src/code/z_demo.c

```diff
@@ -1127,6 +1128,32 @@ void func_800ECD7C(CutsceneContext* csCtx, u8** cutscenePtr, s16 index) {
 
 #ifdef NON_MATCHING
 // Some stack issues, and a few instructions in the "wrong" places
+/**
+ * Loops over the cutscene data itself (`cutscenePtr`), applying the effects of each command instantaneity (for most
+ * commands).
+ *
+ * The cutscene data is a semi-structured array of words, which is made up of
+ * - A beginning, which contains the amount of command lists of this cutscene and the ending frame (see
+ * `CS_BEGIN_CUTSCENE`).
+ * - Any amount of cutscene command lists, which one one contains sub commands of the corresponding type.
```

Comment:
```suggestion
 * - Any amount of cutscene command lists (which should generally be the number specified in the beginning), each one of which contains sub commands of the corresponding type.
```

Reply:
Actually, I probably misunderstood this *again* 😓

Reply:
Yeah, this is right.
But I'm not sure about using "generally"